app/models/inventory_mock.rb

```diff
@@ -59,11 +59,15 @@ def self.delete_item(_uuid)
   end
 
   def self.create_reservation(item_type, start_time, end_time)
-    JSON.parse("{\"id\": \"#{SecureRandom.uuid}\",
-    \"start_time\": \"#{start_time}\",
-    \"end_time\": \"#{end_time}\",
-    \"item_type\": \"#{item_type}\",
-    \"item\": \"Dummy Data\"}")
+    {
+      uuid: SecureRandom.uuid,
+      start_time: start_time,
+      end_time: end_time,
+      item_type: item_type,
+      item: {
+        name: 'Fiore'
+      }
+    }
```

Comment:
turns out that entire class was written by me, i dunno why i couldn't remember that at the time.  also i dont know why I wrote out some json then parsed it rather than just creating a ruby hash directly.